src/main/java/pl/zaprogramujzycie/mma/exceptions/LoginException.java

```diff
@@ -0,0 +1,9 @@
+package pl.zaprogramujzycie.mma.exceptions;
+
+import javax.validation.constraints.NotBlank;
+
+public class LoginException extends RuntimeException {
+    public LoginException(@NotBlank String username, @NotBlank String password) {
+        super("Failed to authenticate with username = %s, password = %s".formatted(username, password));
+    }
```

Comment:
Ja mam exceptions jako podfolder w utils. Trzeba będzie to uspójnić przed mergem.

Reply:
Propnuje wyciagnac poza pakiet utils, ja tak w wiekszosc projektow robie, jesli Ci pasuje to wyciagnij 🗡️

Reply:
Wyjęte